---
std_msgs/Int16 cell_occ_value
std_msgs/Int16 x_size
std_msgs/Int16 y_size
std_msgs/Float32 cell_size
std_msgs/Float32 dilate_radius
